app/assets/stylesheets/hitobito/customizable/_wagon.scss

```
@@ -0,0 +1,58 @@
+//  Copyright (c) 2012-2020, Stiftung für junge Auslandssschweizer. This file is part of
+//  hitobito_sjas and licensed under the Affero General Public License version 3
+//  or later. See the COPYING file at the top-level directory or at
+//  https://github.com/hitobito/hitobito_sjas.
+
+// Links (but not buttons) should be underlined
+a,
+.parent > .nav a:hover {
+  text-decoration: underline;
+}
+a.btn,
+.content-header a {
+  text-decoration: none;
+}
+
+// Uppercase text in buttons and navigation links
+.btn,
+.nav-left-list a,
```

```suggestion
.nav-left-list a,
#page-header a,
```

Ich würde die Links im Header (die jetzt noch weiß auf weiß sind) genauso formatieren.